src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,111 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
```

Comment:
Add index constants and use them in your methods.
```suggestion
@Dao
public class ManufacturerDaoImpl implements ManufacturerDao {
    private static final int NAME_PARAM_INDEX = 1;
    private static final int COUNTRY_PARAM_INDEX = 2;
    private static final int ID_PARAM_INDEX = 3;
  
```